### ISSUE TITLE
How to set Learning rates in Vowpal Wabbit for processing streaming data

### ISSUE THREAD

**Issue by indiaProdip** [JohnLangford/vowpal_wabbit#1178]
I am using Vowpal Wabbit to train a Neural Network on streaming data. What I want is that the learning rate should never go towards 0, else it won't be able to learn from new data. Here are couple of things:
1. If I set power_t to 0 then learning rate won't decay
2. Adaptive gradient accumulates previous gradients per features, so if that is on eventually it will drag learning rate to 0 with time?
3. What role --save_resume plays in this case, does it preserves adaptive learning rate?

In the documentation it says if power_t is 0, it can be used for online tracking, but will not adaptive gradient drag learning update to 0 even if power_t is set to 0.
Thanks,
Prodip

**Comment by JohnLangford** [comment]
When you use --power_t 0, the adagrad part of the learning rule will not be driven to 0.  

--save_resume should preserve _everything_ so that the learning process is identical to simply continuing to process examples rather than doing a save&resume.
